plz.el

```diff
@@ -755,6 +755,7 @@ argument passed to `plz--sentinel', which see."
         (pcase-exhaustive status
           ((or 0 "finished\n")
            ;; Curl exited normally: check HTTP status code.
+           (widen)
```

Comment:
This is a fix for an issue that happens in rare cases. I don't really know why. I mentioned this on the plz PR.
https://github.com/alphapapa/plz.el/pull/43#issuecomment-2016550832